app/lib/s3_util.rb

```diff
@@ -0,0 +1,36 @@
+require "aws-sdk-s3"
+
+module S3Util
+  def self.s3_select_json(bucket, key, expression)
```

Comment:
Can you add a comment "select a particular part of a JSON file using Amazon S3 Select SQL syntax"? To help future users of this function.